example/src/main/AndroidManifest.xml

```diff
@@ -17,5 +17,15 @@
 
 		<activity android:name="com.cube.storm.ui.quiz.activity.StormQuizActivity" android:theme="@style/Theme.AppCompat.Light.DarkActionBar" />
 		<activity android:name="com.cube.storm.ui.quiz.activity.StormQuizResultsActivity" android:theme="@style/Theme.AppCompat.Light.DarkActionBar" />
+
```

Comment:
I think that should be in the library, so we will not need to add that in every app.
I confident it will work.